.github/workflows/promote-to-beta.yml

```diff
@@ -46,9 +46,11 @@ jobs:
             docker manifest push nextcloud/$AIO_NAME\:beta
             
             # arm64
-            docker buildx imagetools create --tag nextcloud/$AIO_NAME\:beta-arm64 nextcloud/$AIO_NAME\:develop-arm64
+            docker manifest create nextcloud/$AIO_NAME\:beta-arm64 -a nextcloud/$AIO_NAME\:develop-arm64
+            docker manifest push nextcloud/$AIO_NAME\:beta-arm64
             
         else
             # amd64
-            docker buildx imagetools create --tag nextcloud/$AIO_NAME\:beta nextcloud/$AIO_NAME\:develop
-        fi
\ No newline at end of file
+            docker manifest create nextcloud/$AIO_NAME\:beta -a nextcloud/$AIO_NAME\:develop
+            docker manifest push nextcloud/$AIO_NAME\:beta

```

Comment:
btw, why did you change this? Shouldn't only the beta-arm64 channel get change or will this not work?

Reply:
I thought the workflow would then be more consistent

Reply:
I would like to keep this for now. I wouldn't touch this part since it seem to work currently.

Reply:
reverted